pkg/workloads/console/controller.go

```diff
@@ -47,6 +47,8 @@ const (
 	ConsoleTemplate = "ConsoleTemplate"
 	Role            = "Role"
 	RoleBinding     = "RoleBinding"
+
+	ConsoleTTL = 24 * time.Hour
```

Comment:
I wonder if instead of defining the TTL as a constant here we could set it in a func passed to `console.Add`? This could solve the test issue potentially

Reply:
That's really sensible. Don't know why I didn't think of that 🤦‍♂️